engine/crates/composition/tests/composition/list_size/subgraphs/a.graphql

```diff
@@ -0,0 +1,11 @@
+type Query {
+  foo: [String!] @listSize(assumedSize: 10)
+  bar(slice: Int!): BarCollection
+    @listSize(slicingArguments: ["slice"], sizedFields: ["length"], requireOneSlicingArgument: false)
```

Comment:
not saying we should validate that right away, but since I'm getting familiar with this directive: shouldn't `sizeFields` references a field that exists on `BarCollection`?

Reply:
Oh yes - I did mean to have it reference a correct field, but fucked up.